logging/impl.go

```diff
@@ -142,33 +143,48 @@ func (imp *impl) log(entry *LogEntry) {
 	}
 }
 
-// Constructs the log message by forwarding to `fmt.Sprint`.
-func (imp *impl) format(logLevel Level, args ...interface{}) *LogEntry {
+// emptyTraceKey is a sentinal value for internal helper functions when there's no context in debug
+// mode.
+const emptyTraceKey = ""
```

Comment:
[opt-nit] `GetName` could probably return `emptyTraceKey` in the case of no trace key to be clear.

Reply:
Done